metrics.md

```diff
@@ -157,12 +157,12 @@ Numerator resource inclusion:
 Notes:
 - Users may add wish to add other elements to this query which can serve as a base
 
-### q_valid_us_core_v4
+### q_valid_us_core_v6
```

Comment:
I figured there's not much value in having both a v4 and v6 test around at the same time? Like, there could be value there, just not sure it's worth the bother.

Reply:
https://github.com/sync-for-science/qualifier/tree/us_core_v4

Reply:
Linked at top now.